doc/files.md

```diff
@@ -81,6 +81,7 @@ Subdirectory | File(s)           | Description
 ## GUI settings
 
 `bitcoin-qt` uses [`QSettings`](https://doc.qt.io/qt-5/qsettings.html) class; this implies platform-specific [locations where application settings are stored](https://doc.qt.io/qt-5/qsettings.html#locations-where-application-settings-are-stored).
+You can change this with the `-guisettings=<path>` option. It will use the cross-platform ini format then. `<path>` is where the `.ini` file is stored.
```

Comment:
I would suggest to call it -guisettings*dir* to be more consistent with other *dir's options we have. Currently, it makes me feel I am expected to write full file path to the `.ini` file, but that's not the case - it's the directory.